package.json

```diff
@@ -71,6 +71,6 @@
     "style-loader": "0.13.1"
   },
   "dependencies": {
-    "@bufferapp/components": "0.1.16"
+    "@bufferapp/components": "0.1.17"
```

Comment:
Ah, good one @stevemdixon 😄 I thought it was okay to update dependencies in the PR - I think it felt more natural since we update it so we can use the new functionality in buffer-components. Happy to do it in a separate branch if that feels better for the flow here 😄